docs/audit/nitpicker-findings.md

```
@@ -49,6 +29,16 @@ The repo no longer references it, so it won't be reinstalled.
 
 ## Fixed
 
+### Pass 3 — 2026-04-27
+
+#### [N-010] No verification scenario for fish-without-tmux (wezterm direct)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)